tests/specs/benchmarks/verification.k

```diff
@@ -307,12 +311,37 @@ module VERIFICATION-HASKELL [symbolic, kore]
 
     rule #gas(G) -Int G' => #gas(G +Int G') [simplification]
 
+    rule #gas(_)  <Int I => false requires I <=Int pow256 [simplification]
```

Comment:
Put this and all related into a new section "Infinite gas assumption"

Reply:
I'm not sure it makes sense to do this. We currently have infinite gas assumption for Java backend, it's just done using the modified `#deductGas` rules instead of using this technique, so I don't see why it's different.

These are also the only lemmas existing in the `VERIFICATION-HASKELL`, so I'm not sure a separate section makes sense. And this is only in the `benchmarks/verification.k`, so it's not affecting any other proofs.

Reply:
It's not particularly important, but it doesn't hurt adding a comment explaining the reason for these lemmas. This sort of comments is one of the documentation forms lacking in RV. But I won't hold the review on this. Just a recommendation.

Reply:
Ah ok, I thought you meant a separate module. I've added a comment explaining how it works, and the benefits.